Lombiq.Gulp.Extensions.csproj

```diff
@@ -26,7 +26,21 @@
     - Binding InitCommonNodeModules to run before both of those ensures that the NPM-related commands will run towards the
       beginning of the build only once, so there are no concurrent processes locking each other's files.
   -->
-  <Target Name="InitCommonNodeModules" BeforeTargets="ResolveReferences" Inputs="package.json" Outputs="$(InitCommonNodeModulesStampFile)">
+  <Target Name="InitCommonNodeModules"
+          AfterTargets="EnablePnpm"
+          BeforeTargets="ResolveReferences"
+          Inputs="package.json"
+          Outputs="$(InitCommonNodeModulesStampFile)">
+    <!-- Ignore sporadic errors when this is run by multiple threads at the same time. -->
+    <Exec Command="corepack enable"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="$(NxVerbosity)" />
+    <!-- Enable the latest version of pnpm@7. -->
+    <Exec Command="corepack prepare pnpm@latest-7 --activate"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="$(NxVerbosity)" />
```

Comment:
I can't find a better place for this.

Reply:
![image](https://user-images.githubusercontent.com/164360/228540361-5f312d98-48b0-4fb9-ae9d-5b44101168dc.png)